.github/workflows/static-analyze.yml

```diff
@@ -19,7 +19,9 @@ jobs:
         dependencies:
           - "locked"
         php-version:
-          - "7.4"
+          - "8.1"
+          - "8.2"
+          - "8.3"
```

Comment:
oh btw. static analyze should be only executed at 8.1, sorry I missed that. Tests at all supported PHP versions, static analyze and other "development" processes at the lowest supported version